suite/tests/linux/clone.c

```diff
@@ -104,6 +104,8 @@ static void *stack;
 void
```

Comment:
Re the title starting with `iX`: re-iterating Derek's comment: https://github.com/DynamoRIO/dynamorio/pull/6515#discussion_r1431775992.

Reply:
Ack